src/pages/StakingPageSingle.svelte

```diff
@@ -445,7 +378,7 @@ metadata={{
               <div class="flex items-center justify-between">
                 <div class="flex nowrap intems-center p-1 font-thin">Amount to Stake</div>
                 <div class="right text-white font-bold text-xs py-1px text-center align-right float-right rounded">
-                  <button on:click={() => { stakeAmount = formatEther(data.userTokenBalance)}} class="oven-withdraw-button">100%</button>
+                  <button disabled on:click={() => { stakeAmount = formatEther(data.userTokenBalance)}} class="oven-withdraw-button">100%</button>
```

Comment:
Why have the button here in the first place? If it's permanently disabled why not have a UI element that indicates staking is perma-disabled

Reply:
That's why the button text was changed